src/link/error/index.ts

```diff
@@ -8,6 +16,10 @@ import { ApolloLink } from "../core/index.js";
 export interface ErrorResponse {
   graphQLErrors?: ReadonlyArray<GraphQLFormattedError>;
   networkError?: NetworkError;
+  protocolErrors?: ReadonlyArray<{
```

Comment:
I know we call those "protocol errors" internally, but looking at when these actually occur, could we maybe call this `incrementalErrors` instead?
```suggestion
  incrementalErrors?: ReadonlyArray<{
```

Reply:
I named this `protocolErrors` to match the same property used in [`ApolloError`](https://github.com/apollographql/apollo-client/blob/6ffb5a34a39657568db7f4e42ed60d8679cb817f/src/errors/index.ts#L25-L28) where these specific types of errors are [stored](https://github.com/apollographql/apollo-client/blob/6ffb5a34a39657568db7f4e42ed60d8679cb817f/src/core/QueryManager.ts#L1065-L1067).

These errors aren't actually incremental errors but rather the [transport-level errors](https://www.apollographql.com/docs/graphos/routing/operations/subscriptions/multipart-protocol#message-and-error-format) sent from the router that occur when it can't communicate with the subgraph (i.e. the `errors` field next to `payload`, but not the `errors` field inside the `payload` property, which are just `graphQLErrors`).